src/index.js

```diff
@@ -38,16 +38,37 @@ function PaymentsClient(config) {
   if (this.httpsOnly === false) {
     console.warn('httpsOnly is set to false. Only use for dev');
   }
+
+  var that = this;
+  window.addEventListener('message', function(e) {
+    that.receiveMessage.call(that, e);
+  }, false);
+
   return this;
 }
 
 PaymentsClient.prototype = {
+
+  validIframeOrigins: [
+    'http://pay.dev:8000',
+    'http://pay.dev.mozaws.net:8000',
```

Comment:
this should probably be configurable in global settings but we could always add that when needed